myriaX.markdown

```diff
@@ -7,152 +7,132 @@ weight: 1
 
 # MyriaX Engine
 
-#Preparing to use
+## Local Installation
 
-#### 0. Build the Myria jar
-In Myria source (`../`), run `./gradlew jar`.
+### 0. Preparation
 
-#### 1. Myria needs Java 7
-Make sure `java -version` shows 7 on any machines in the cluster. 
+#### Myria needs Java 7
```

Comment:
I think this should be "Java 7 or later" -- Java 8 came out since we wrote these instructions, and Myria works with it.